libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/internal/MultipartReader.kt

```diff
@@ -110,6 +110,14 @@ class MultipartReader constructor(
       when (source.select(afterBoundaryOptions)) {
         0 -> {
           // "\r\n": We've found a new part.
+          if (source.select(Options.of("--$boundary--".encodeUtf8())) == 0) {
```

Comment:
Can this be done at the same time as the other select? Looks like creating `Options` [could be expensive](https://medium.com/@jerzy.chalupski/okio-options-ce8f3ac1584f) so I wouldn't want to create too many?